lib/vsc/utils/py2vs3/py2.py

```diff
@@ -41,6 +42,7 @@
 FileExistsErrorExc = OSError  # noqa
 FileNotFoundErrorExc = OSError  # noqa
 
+logging.warning("py2 module deprecated, please stop using it.")
```

Comment:
I left it in because I did not remove the whole py2vs3 thing, I can however remove it, but it might break in other packages. I was thinking about removeing the whole py2vs3 in one go at a later stage, when more cleanups have happened.  I'll make it a raise.

Reply:
hmm, I might indeed just remove it and raise the exception in ```py2vs3/__init__.py```